CONTRIBUTING.md

```diff
@@ -25,6 +32,13 @@ Please be clear about your requirements. Incomplete feature requests may simply
 be closed if we don't understand what you would like to see added to Flux.
 
 ## Contributing to the source code
+
+### Finding an issue
+The Flux team regularly adds the [community](https://github.com/influxdata/flux/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity) label to issues that we think would be accessible for a community member to take.
+Before starting to work on an issue, please inform us on GitHub about your intention to work on the issue.
```

Comment:
```suggestion
Before starting to work on an issue, please inform us on GitHub about your intention to work on the issue by leaving a comment on the issue and assigning it to yourself.
```

Reply:
I actually don't think they can assign it to themselves. I'll have to fix this.